content/en/blog/_posts/xxxx-xx-xx-Fine-grained-SupplementalGroups-control/index.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: blog
+title: 'Kubernetes 1.31: Fine-grained SupplementalGroups control'
+date: xxxx-xx-xx
+slug: fine-grained-supplementalgroups-control
+author: >
+  Shingo Omura (Woven By Toyota)
+
+---
+
+This blog discusses a new feature in Kubernetes 1.31 to improve the handling of supplementary groups in containers in Pods.
+
+
+## Motivation: Implicit group memberships defined in `/etc/group` in the container image
+
+Although this behavior may not be quite popular by many Kubernetes cluster users/admins, kubernetes, by default, _merges_ group information from the Pod with information defined in `/etc/group` in the container image.
+
+Let's see an example, below Pod specifies `runAsUser=1000`, `runAsGroup=3000` and `supplementalGroups=4000` in the Pod's security context.
+
+{{% code_sample file="implicit-groups.yaml" %}}
+
+What is the result of `id` command in the `ctr` container?
+
+```shell
+# Create the Pod:
+$ kubectl apply -f https://k8s.io/blog/xxxx-xx-xx-Fine-grained-SupplementalGroups-control/implicit-groups.yaml
```

Comment:
Insert date in the kubectl command

Reply:
fixed in https://github.com/kubernetes/website/pull/46921/commits/14612bb32776a7b740db3adddfa8600f6a6760f4